hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/analysis/HoodieAnalysis.scala

```diff
@@ -268,7 +269,25 @@ case class HoodieResolveReferences(sparkSession: SparkSession) extends Rule[Logi
       } else {
         l
       }
-
+    // Fill schema for Create Table without specify schema info
+    case c @ CreateTable(tableDesc, _, _)
+      if isHoodieTable(tableDesc) =>
+        val tablePath = getTableLocation(c.tableDesc, sparkSession)
+          .getOrElse(s"Missing location defined in table ${c.tableDesc.identifier}")
```

Comment:
is it that for a new table thats getting created, tablePath will be set to "Missing location defined in table ..." ? In other words, if table already existed, tablePath will be set to right value, if not, it will be set to this string. 
Is my understanding right? If so, why can't we return right away if there table does not exist to maintain the same flow as before for tables that are just getting created.

Reply:
Or will this case c @ CreateTable(tableDesc, _, _) will be invoked only if table already exists?

Reply:
> is it that for a new table thats getting created, tablePath will be set to "Missing location defined in table ..." ? In other words, if table already existed, tablePath will be set to right value, if not, it will be set to this string.
> Is my understanding right? If so, why can't we return right away if there table does not exist to maintain the same flow as before for tables that are just getting created.

My mistake! I want to throw an Exception here. Will fix it soon.